### DISCUSSION TITLE
Deploying with GitHub Pages

### DISCUSSION THREAD

**Discussion by kakusu243** [microsoft/SATechnicalOnboarding#106] [Questions]
blank page when viewing my GitHub page. Please help me!!
!
![Capture d'écran 2024-02-04 154157](https://github.com/microsoft/SATechnicalOnboarding/assets/138121099/13c9e610-42b1-4fa4-a100-e04aa320e010)

**Comment by AnthonyByansi** [comment]
hi @kakusu243 from what you sent, it indicates your site is published to Github pages. Please note that you are not meant to use `github pages`  to deploy the portfolio site. You're required to deploy your site on `Azure` instead. 

Please start off here: https://github.com/education/codespaces-project-template-js
